apps/src/lib/ui/SettingsCog.jsx

```diff
@@ -135,6 +140,9 @@ class SettingsCog extends Component {
       rootStyle.color = color.dark_charcoal;
     }
 
+    const aiEnabled =
+      experiments.isEnabled(experiments.APPLAB_ML) && this.areAIToolsEnabled();
```

Comment:
this will check `localStorage` every time this component renders, which can cause performance issues, so we usually handle this by checking if the experiment is enabled in `componentDidMount` and setting that boolean in component state